src/http.rs

```diff
@@ -56,6 +56,10 @@ pub enum HttpError {
     CallRequest { source: hyper::Error },
     #[snafu(display("Failed to build HTTP request: {}", source))]
     BuildRequest { source: http::Error },
+    #[snafu(display("Failed to acquire authentication resource."))]
+    ExtensionAuthentication {
```

Comment:
Nit: rename to `AuthenticationExtension`.

Reply:
fixed